Branch-SDK/Branch-SDK/BNCDeviceInfo.m

```diff
@@ -124,7 +125,7 @@ + (NSString*) userAgentString {
             BNCPreferenceHelper *preferences = [BNCPreferenceHelper preferenceHelper];
             preferences.browserUserAgentString = browserUserAgentString;
             preferences.lastSystemBuildVersion = self.systemBuildVersion;
-			//NSLog(@"[Branch] userAgentString: '%@'.", browserUserAgentString);
```

Comment:
Some of the debug message are useful for debugging the SDK and some are useful for external developers who are integrating Branch and debugging their own apps.

I can add another log level for internal use:  `BNCLogLevelDebugSDK`.